app/code/Magento/Persistent/Test/Unit/CustomerData/PersistentTest.php

```diff
@@ -0,0 +1,109 @@
+use Magento\Persistent\CustomerData\Persistent;
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Customer\Helper\View;
+use Magento\Persistent\Helper\Session;
+use Magento\Persistent\Model\Session as PersistentSession;
+use Magento\Customer\Api\Data\CustomerInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+
+class PersistentTest extends TestCase
+{
+    /**
+     * @var Persistent
+     */
+    private $customerData;
+
+    /**
+     * @var Session|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $persistentSessionHelperMock;
+
+    /**
+     * @var View|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $customerViewHelperMock;
+
+    /**
+     * @var CustomerRepositoryInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $customerRepositoryMock;
+
+    /**
+     * Setup environment for test
+     */
+    protected function setUp()
+    {
+        $this->persistentSessionHelperMock = $this->createMock(Session::class);
+        $this->customerViewHelperMock = $this->createMock(View::class);
+        $this->customerRepositoryMock = $this->createMock(CustomerRepositoryInterface::class);
+
+        $objectManager = new ObjectManagerHelper($this);
+
+        $this->customerData = $objectManager->getObject(
+            Persistent::class,
+            [
+                'persistentSession' => $this->persistentSessionHelperMock,
+                'customerViewHelper' => $this->customerViewHelperMock,
+                'customerRepository' => $this->customerRepositoryMock
+            ]
+        );
+    }
+
+    /**
+     * Test getSectionData() when disable persistent
+     */
+    public function testGetSectionDataWhenDisablePersistent()
+    {
+        $this->persistentSessionHelperMock->method('isPersistent')->willReturn(false);
+
+        $this->assertEquals([], $this->customerData->getSectionData());
+    }
+
+    /**
+     * Test getSectionData() when customer doesn't login
+     */
+    public function testGetSectionDataWithNotLogin()
+    {
+        $this->persistentSessionHelperMock->method('isPersistent')->willReturn(true);
+
+        $persistentSessionMock = $this->createPartialMock(PersistentSession::class, ['getCustomerId']);
+        $persistentSessionMock->method('getCustomerId')->willReturn(null);
+        $this->persistentSessionHelperMock->method('getSession')->willReturn($persistentSessionMock);
+
+        $this->assertEquals([], $this->customerData->getSectionData());
+    }
+
+    /**
+     * Test getSectionData() when customer login and enable persistent
+     */
+    public function testGetSectionDataCustomerLoginAndEnablePersistent()
+    {
+        $this->persistentSessionHelperMock->method('isPersistent')->willReturn(true);
+
+        $persistentSessionMock = $this->createPartialMock(PersistentSession::class, ['getCustomerId']);
+        $persistentSessionMock->method('getCustomerId')->willReturn(1);
```

Comment:
You should not use magic numbers - that should be private const with a meaningful name - like `STUB_CUSTOMER_ID`